resources/definitions/fdmprinter.def.json

```diff
@@ -4565,7 +4565,17 @@
                     "type": "bool",
                     "default_value": false,
                     "settable_per_mesh": false,
-                    "settable_per_extruder": false
+                    "settable_per_extruder": false,
+                    "children":
+                    {
+                        "smooth_spiralized_contours":
```

Comment:
Well, sorry, I did not understand this requirement. To be honest, I find the whole Cura UI programming & configuration scheme completely baffling. Could some kind person please fix this for me. Thanks, Mark